Sources_v3/StreamChatUI/Utils/UIFont+Extensions.swift

```diff
@@ -10,7 +10,7 @@ extension UIFont {
         return UIFont(descriptor: descriptor!, size: pointSize)
     }
 
-    func bold() -> UIFont {
+    var bold: UIFont {
```

Comment:
Inside you create new font instance based on font description changes. This operation can be who nows how time consuming. 

By making it computed property you states that this operation is lightweight (property access lightweight). That can be misleading.

IMHO this is regression.

Reply:
I find it convenient to use computed properties instead of functions with no input params. That's a common practice to have a property which creates some instance of another type based on `self` properties instead of introducing new init for the target type (or same type).

Do you have strong opinion on that? 🙂 If yes - let's switch to functions then